fix llama-runner-linux-android

manuelcandales opened the pull request:
Differential Revision: D56306284

larryliu0820 commented on .ci/scripts/build_llama_android.sh:
```diff
@@ -42,6 +42,7 @@ build_llama_runner() {
     -DCMAKE_INSTALL_PREFIX=cmake-android-out \
     -DCMAKE_BUILD_TYPE=Release -DPYTHON_EXECUTABLE=python \
     -DEXECUTORCH_BUILD_OPTIMIZED=ON \
+    -DEXECUTORCH_BUILD_QUANTIZED=ON \
```
Sorry not here

larryliu0820 replied:
Can you add it to line 28-29

manuelcandales committed:
fix llama-runner-linux-android (#3127)

Summary: Pull Request resolved: https://github.com/pytorch/executorch/pull/3127

Reviewed By: larryliu0820, kirklandsign

Differential Revision: D56306284
